amundsen_application/api/metadata/v0.py

```diff
@@ -494,3 +472,68 @@ def _log_get_user(*, user_id: str) -> None:
         logging.exception(message)
         payload = jsonify({'msg': message})
         return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+@metadata_blueprint.route('/user/bookmark', methods=['GET'])
+def get_bookmark() -> Response:
+    try:
+        user_id = request.args.get('user_id')
+        if user_id is None:
+            if app.config['AUTH_USER_METHOD']:
+                user_id = app.config['AUTH_USER_METHOD'](app).user_id
+            else:
+                raise Exception('AUTH_USER_METHOD is not configured')
+
+        url = '{0}{1}/{2}/follow/'.format(app.config['METADATASERVICE_BASE'], USER_ENDPOINT, user_id)
+
+        response = request_metadata(url=url, method=request.method)
+        status_code = response.status_code
+
+        tables = response.json().get('table')
+        table_bookmarks = [marshall_table_partial(table) for table in tables]
+
+        payload = {
+            'msg': 'success',
+            'bookmarks': table_bookmarks
+        }
+
+        return make_response(jsonify(payload), status_code)
+    except Exception as e:
+        message = 'Encountered exception: ' + str(e)
+        logging.exception(message)
+        payload = jsonify({'msg': message})
+        return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+@metadata_blueprint.route('/user/bookmark', methods=['PUT', 'DELETE'])
+def update_bookmark() -> Response:
+    try:
+        if app.config['AUTH_USER_METHOD']:
+            user = app.config['AUTH_USER_METHOD'](app)
+        else:
+            raise Exception('AUTH_USER_METHOD is not configured')
+
+        args = request.get_json()
+        resource_type = get_query_param(args, 'type')
+        resource_key = get_query_param(args, 'key')
+
+        url = '{0}{1}/{2}/follow/{3}/{4}'.format(app.config['METADATASERVICE_BASE'],
+                                                 USER_ENDPOINT,
+                                                 user.user_id,
+                                                 resource_type,
+                                                 resource_key)
+
+        response = request_metadata(url=url, method=request.method)
+        status_code = response.status_code
+
+        payload = {
+            'msg': 'success',
+            'response': response.json()
+        }
+
+        return make_response(jsonify(payload), status_code)
+    except Exception as e:
+        message = 'Encountered exception: ' + str(e)
```

Comment:
could we just combine the two lines into single line? We normally just put the payload directly into the same line instead of creating an additional variable for that.